Backend/app/rider-platform/rider-app/Main/src/Beckn/ACL/OnUpdate.hs

```diff
@@ -60,6 +60,7 @@ parseEvent _ (OnUpdate.RideAssigned taEvent) = do
   agent <- fromMaybeM (InvalidRequest "agent is not present in RideAssigned Event.") $ taEvent.fulfillment.agent
   agentPhone <- fromMaybeM (InvalidRequest "agent phoneNumber is not present in RideAssigned Event.") $ agent.phone
   tagsGroup <- fromMaybeM (InvalidRequest "agent tags is not present in RideAssigned Event.") agent.tags
+  driverInitialLocation <- fromMaybeM (InvalidRequest "Driver Initial Location is not present in RideAssigned Event.") $ taEvent.fulfillment.driverInitialLocation
```

Comment:
Why to throw error if `driverInitialLocation` is not there. We discussed this on last reliability call, right? We should not throw error in critical API for non critical data.